pyke/lightcurve.py

```diff
@@ -31,6 +31,10 @@ def __init__(self, time, flux, flux_err=None):
         self.time = time
         self.flux = flux
         self.flux_err = flux_err
+        # If the data was given as a list, convert it into a NumPy array
+        for attr in ['time', 'flux', 'flux_err']:
+            if isinstance(getattr(self, attr), list):
+                setattr(self, attr, np.array(getattr(self, attr)))
```

Comment:
My unit test failed initially because I was passing `time` and `flux` as lists to a `LightCurve` object.  I thought it would be good to support lists, so I changed the constructor to do the conversion to numpy arrays when lists are given. Do you think this is the right thing to do?

Reply:
let's just do
```
self.time = np.asarray(time)
self.flux = np.asarray(flux)
self.flux_err = np.asarray(flux_err)
```